dts/arm64/intel/intel_socfpga_agilex.dtsi

```diff
@@ -99,6 +99,10 @@
 		reg = <0x10000000 0x200000>;
 	};
 
+	fpga0: bridges {
```

Comment:
Shouldn't there be an address associated with this device inside of the fpga SoC?

Reply:
Actaully we are not directly interacting with any hardware we are calling sip_svc subsystem which further request the ATF to read and write in the bridge register so this bridge dont have any address